### DISCUSSION TITLE
Trilium Backup problem

### DISCUSSION THREAD

**Discussion by xsmyqf** [TriliumNext/Trilium#3317] [General]
Will Trilium auto automatically copy backup-daily ,backup-now, backup-weekly to somewhere? Can I find last week's backup?

**Comment by xsmyqf** [comment]
I want to restore some week in last month, Is this possible?

**Comment by xsmyqf** [comment]
how to restore recovered files?

**Comment by sigaloid** [comment]
Trilium will back up to the backup folder. https://github.com/zadam/trilium/wiki/Backup

**Reply by xsmyqf** [reply]
![image](https://user-images.githubusercontent.com/6007829/201940042-4aff2066-984e-42b6-bc5e-7f257d031799.png)
this folder is only for the latest backup, is there a folder which contain history backup?

**Reply by xsmyqf** [reply]
I find a folder tagged with recovered branch lost its content,I want to recover it.

**Reply by sigaloid** [reply]
No, if there is no older ones, then trilium doesn't have older ones

**Reply by xsmyqf** [reply]
So, I can not recover these files if they are deleted or recovered?

**Comment by zadam** [comment]
The guide to restore the backup is here: https://github.com/zadam/trilium/wiki/Backup#restoring-backup
